tests/MongoDB.Driver.Tests/Specifications/Runner/MongoClientJsonDrivenTestRunnerBase.cs

```diff
@@ -356,10 +349,29 @@ protected void SetupAndRunTest(JsonDrivenTestCase testCase)
             SetupAndRunTest(testCase.Shared, testCase.Test);
         }
 
-        protected void VerifyCollectionData(IEnumerable<BsonDocument> expectedDocuments, Action<BsonDocument, BsonDocument> prepareExpectedResult)
+        protected virtual void VerifyCollectionData(
+            IEnumerable<BsonDocument> expectedDocuments,
+            string databaseName,
+            string collectionName,
+            ReadConcern readConcern = null)
+        {
+            VerifyCollectionData(expectedDocuments, databaseName, collectionName, readConcern: readConcern, prepareExpectedResult: null);
+        }
+
+        protected void VerifyCollectionData(
+            IEnumerable<BsonDocument> expectedDocuments,
+            string databaseName,
+            string collectionName,
+            ReadConcern readConcern,
```

Comment:
readConcern not needed. Just use `ReadConcern.Default` for all verification of collection data.

Reply:
done